src/screens/sharing/Sharing.tsx

```diff
@@ -53,23 +59,23 @@ export const SharingScreen = () => {
           navLabel={i18n.translate('Sharing.Close')}
           onIconClicked={close}
         />
-        <ScrollView>
```

Comment:
Think we need to keep ScrollView here too for accessibility

Reply:
Added it back!